src/open_inwoner/scss/components/Typography/H2.scss

```diff
@@ -15,6 +14,10 @@
     overflow-wrap: break-word;
   }
 
+  .link {
+    font-size: var(--utrecht-heading-2-font-size);
+  }
+
```

Comment:
For regular links we have `text-decoration: none;` and then `text-decoration: underline;` on `hover`. On the other hand: `h2` headings which are also links are always underlined. Do we want to treat the headings differently or should we make them consistent with regular links?

Reply:
Good question. We don't have a design for this, but it's actually better for accessibility to make them underlined first, which is why I had to make links in the FAQ and product pages underlined. Surprisingly none of the accessibility audits asked us to change/reverse all of our anchor links this way. :astonished:
I'd say: let's keep headings conform to the overall style first, so it looks better on this page. If in future I'll need to adjust all link styles everywhere, that'd be a new issue.